### DISCUSSION TITLE
Why not replace Redux?

### DISCUSSION THREAD

**Discussion by NodeGuy** [crimx/observable-hooks#52] [Q&A]
From the README:

`This library is not for replacing state management tools like Redux but to reduce the need of dumping everything into global state.`

Replacing Redux with this sounds like a great idea to me; why do you recommend against it?

**Comment by crimx** [comment]
Hi, I don't recommend against it. I am saying this library is not for this purpose. In fact `useObservableState` has been re-designed to make it work with reducer pattern.

What Redux offers are much more than that. Tool chains, ecosystems, docs and tutorials. It needs to be another library for this.
